### ISSUE TITLE
Create DT::datatable of all current indicators in Fingertips by profile and area

### ISSUE THREAD

**Issue by julianflowers** [PublicHealthEngland/fingertipsR#34]
A function to create a data table of live indicators  - which profile and what geography

`
indicator_List<- function(){
require(tidyverse)

fingertipsR::areatypes_by_indicators %>%
  left_join(area_types()) %>%
  group_by(IndicatorID) %>%
  select(IndicatorID, AreaTypeName) %>%
  arrange(IndicatorID) %>%
  left_join(indicators()) %>%
  select(IndicatorID, IndicatorName, ProfileName, AreaTypeName) %>%
  distinct() %>%
  DT::datatable(filter = "top", extensions = 'Buttons', options = list(
    dom = 'Bfrtip',
    buttons = c('copy', 'csv', 'excel', 'pdf', 'print')
  )
)
}
`

**Comment by sebastian-fox** [comment]
Thanks for the suggestion. What do you think is the purpose of this function (eg, how do you think the output should be presented?):

* A data.frame object?
* a Shiny gadget (like `select_indicators()`)? So people can point and click to retrieve what they need
* other?

My thought is it should be a data.frame (which would mean we miss off the last few lines of your code).
Agree?

**Comment by julianflowers** [comment]
Yes I think you are right a data frame - i think we need something which shows which geography is availalbe for which indicator
I kind of like the fact that you can download data from dt::datatable though
